transactions/set-version-boundary/2025/may-15/arguments.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "type": "UInt8",
+    "value": "0"
+  },
+  {
+    "type": "UInt8",
+    "value": "41"
+  },
+  {
+    "type": "UInt8",
+    "value": "0"
+  },
+  {
+    "type": "String",
+    "value": ""
```

Comment:
```suggestion
    "value": null
```

Reply:
updated to null. Check the latest version. Thanks for flagging it.